source/hiring/index.md

```diff
@@ -1,90 +1,30 @@
 # arXiv is hiring!
 
-We are looking for a community engagement manager, a few good developers, a program manager, a tech writer/documentation specialist and more.
+We are presently looking for a [Production Editor](#production-editor).
 
  - arXiv is part of Cornell University. All arXiv employees are Cornell University employees.
  - The positions listed on this page are all full time [staff positions with benefits (endowed)](https://hr.cornell.edu/understand-your-benefits).
  - Visa sponsorship is not available for these positions.
  - Unless otherwise stated, these are 3-year renewable appointments
```

Comment:
```suggestion
```

I'd remove this as I prefer to talk this through in context with applicants.

Reply:
I have removed the line on my local branch.